at_client/lib/src/client/at_client_impl.dart

```diff
@@ -493,121 +273,40 @@ class AtClientImpl implements AtClient {
-      {String? sharedWith, Metadata? metadata}) async {
-    var updateKey = key;
-    if (metadata == null || metadata.namespaceAware) {
-      updateKey = _getKeyWithNamespace(key);
-    }
-    var operation = getOperation(value, metadata);
-    sharedWith = AtUtils.formatAtSign(sharedWith);
-    var builder = UpdateVerbBuilder()
-      ..atKey = updateKey
-      ..sharedBy = currentAtSign
-      ..sharedWith = sharedWith
-      ..value = value
-      ..operation = operation;
-
-    if (metadata != null) {
-      builder.ttl = metadata.ttl;
-      builder.ttb = metadata.ttb;
-      builder.ttr = metadata.ttr;
-      builder.ccd = metadata.ccd;
-      builder.isBinary = metadata.isBinary;
-      builder.isEncrypted = metadata.isEncrypted;
-      builder.isPublic = metadata.isPublic!;
-      if (metadata.isHidden) {
-        builder.atKey = '_' + updateKey;
-      }
-    }
-    if (value != null) {
-      if (sharedWith != null && sharedWith != currentAtSign) {
-        try {
-          builder.value =
-              await _encryptionService!.encrypt(key, value, sharedWith);
-          builder.isEncrypted = true;
-        } on KeyNotFoundException catch (e) {
-          var errorCode = AtClientExceptionUtil.getErrorCode(e);
-          return Future.error(AtClientException(errorCode, e.message));
-        }
-      } else if (!builder.isPublic &&
-          !builder.atKey.toString().startsWith('_')) {
-        builder.value = await _encryptionService!.encryptForSelf(key, value);
-        builder.isEncrypted = true;
-      }
-    }
-    var isSyncRequired = true;
-    if (updateKey.startsWith(AT_PKAM_PRIVATE_KEY) ||
-        updateKey.startsWith(AT_PKAM_PUBLIC_KEY)) {
-      builder.sharedBy = null;
-    }
-    if (SyncUtil.shouldSkipSync(updateKey)) {
-      isSyncRequired = false;
-    }
-    //sign public data with private encryption key
-    if (metadata != null && metadata.isPublic!) {
-      try {
-        var encryptionPrivateKey =
-            await _localSecondary!.getEncryptionPrivateKey();
-        if (encryptionPrivateKey != null) {
-          _logger.finer('signing public data for key:$key');
-          builder.dataSignature =
-              _encryptionService!.signPublicData(encryptionPrivateKey, value);
-        }
-      } on Exception catch (e) {
-        _logger.severe('Exception trying to sign public data:${e.toString()}');
-      }
-    }
-
-    String? putResult;
-    try {
-      if (builder.dataSignature != null) {
-        builder.isJson = true;
-      }
-      putResult =
-          await getSecondary().executeVerb(builder, sync: isSyncRequired);
-    } on AtClientException catch (e) {
-      _logger.severe(
-          'error code: ${e.errorCode} error message: ${e.errorMessage}');
-    } on Exception catch (e) {
-      _logger.severe('error in put: ${e.toString()}');
-    }
-    return putResult != null;
+  @override
+  Future<AtValue> get(AtKey atKey, {bool isDedicated = false}) async {
```

Comment:
Huge improvement in readability simplicity and maintainability between the previous and new versions of get(..) and put(..) ... great work